includes/class-migration.php

```diff
@@ -203,6 +199,10 @@ public static function maybe_migrate() {
 			}
 		}
 
+		// Always register schedules, to ensure they are registered even
+		// if the plugin is activated after the first activation.
```

Comment:
```suggestion
		// Ensure all required cron schedules are registered.
```
This might be all that's needed in terms of context?

Reading "ensure they are registered even if the plugin is activated after the first activation" makes me wonder what's happening there, what's going wrong, what needs fixing, if that makes sense?

It's probably fine for us to just keep this as a fallback. If it's to fix a bug that we've not investigated, we should investigate it and fix it 🙂

Reply:
That case have so many reasons! I also accidentally removed too many schedules, because of some were stuck! I would not assume a bug here! The logic is quite simple and I see nothing else that might delete a schedule.